src/test/kotlin/com/target/liteforjdbc/ConnectionSessionTest.kt

```diff
@@ -312,6 +312,70 @@ class ConnectionSessionTest {
         confirmVerified(mockAutoCommitConnection, mockInsertPreparedStatement)
     }
 
+
+    @Test
+    fun testExecuteBatchWithGeneratedKeys() {
+        val namedSql = "INSERT INTO TABLE (FIELD, FIELD2) VALUES (:fieldVal, :field2Val)"
+
+        val result = autoCommit.executeBatch(
+            namedSql,
+            listOf(
+                mapOf("fieldVal" to "value", "field2Val" to 1),
+                mapOf("fieldVal" to "value2", "field2Val" to 2)
+            )
+        ) { resultSet -> resultSet.getInt("id") }
+
+        result.size shouldBe 2
+        result[0] shouldBe 10
```

Comment:
Maybe a future PR, but it'd be nice if these hard-coded values were constants...I had to look around to see where they were coming from.  Also there's a typo in `mockGeneratredKeysResultSet`